docs/tutorials/09_Protein_Folding.ipynb

```diff
@@ -0,0 +1,824 @@
+    "\n",
+    "where \n",
+    "\n",
+    "- $H_{gc}$ is the geometrical constraint term (governing the growth of the primary sequence of aminoacids without bifurcations\n",
+    "\n",
+    "- $H_{ch}$ is the chirality constraint (enforcing the right stereochemistry for the system)\n",
+    "\n",
+    "- $H_{in}$ is the interaction energy terms of the system. In our case we consider only nearest neighbor interactions. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "62d99a5f",
+   "metadata": {},
+   "source": [
+    "Further details about the used model and the encoding of the problem can be found in [2]."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "8f541d04",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from qiskit_nature.problems.sampling.protein_folding.interactions.random_interaction import RandomInteraction \n",
+    "from qiskit_nature.problems.sampling.protein_folding.interactions.miyazawa_jernigan_interaction import MiyazawaJerniganInteraction \n",
+    "from qiskit_nature.problems.sampling.protein_folding.peptide.peptide import Peptide\n",
+    "from qiskit_nature.problems.sampling.protein_folding.protein_folding_problem import ProteinFoldingProblem\n",
+    "\n",
+    "from qiskit_nature.problems.sampling.protein_folding.penalty_parameters import PenaltyParameters\n",
+    "\n",
+    "from qiskit.utils import algorithm_globals, QuantumInstance\n",
+    "\n",
+    "algorithm_globals.random_seed = 23"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fced9189",
+   "metadata": {},
+   "source": [
+    "### Protein Main Chain"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "350c031c",
+   "metadata": {},
+   "source": [
+    "The Protein consists of a main chain that is a linear chain of aminoacids. For the naming of different residues we use the one-letter code as defined in Ref. [3]. Further details about the naming and the type of aminoacids can also be found in [4].\n",
+    "\n",
+    "For this particular case we demonstrate the generation of the qubit operator in a neuropeptide with the main chain consisting of 7 aminoacids with letter codes APRLRFY (see also [2])."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "bf140d37",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "main_chain = \"APRLRFY\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e399913f",
+   "metadata": {},
+   "source": [
+    "### Side Chains"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d71b4571",
+   "metadata": {},
+   "source": [
+    "Beyond the main chain of the protein there may be aminoacids attached to the residues of the main chain. Our model allows for side chains of the maximum length of one. Elongated side chains would require the introduction of additional penalty terms which are still under development. "
```

Comment:
Maybe add a sentence that in this simple example we don't specify any side chains. Otherwise the following code block is not really connected to the text.